docs/contributing/index.rst

```diff
@@ -3,59 +3,32 @@
 Contributing to Weblate
 =======================
 
+Language
+--------
 
-There are dozens of ways to contribute in Weblate. Any help is welcomed, be it
```

Comment:
Two sentences doing what the sidebar does.
Then a table of contents doing it yet again.
Then (if there wasn't so much text), headlines that can actually be browsed to that effect. Then "in depth" guide links making it all hollow.

Make sure the first try sticks, and then get fancy.

Reply:
Reader should be able to follow the documentation from the text. The sidebar is nice, but hardly usable on mobile and having links in the text is way easier to navigate in this case.

Compare:
![image](https://user-images.githubusercontent.com/212189/111788616-93d15500-88c0-11eb-8502-f19672cb9b2c.png) ![image](https://user-images.githubusercontent.com/212189/111788646-9b90f980-88c0-11eb-9376-b7b168c1b0e7.png)